members/CusB729dc0101592930443.yaml

```diff
@@ -108,9 +109,8 @@ contact_form:
     - click_on:
       - selector: "#comment-view-page button.btn.btn-primary"
     - find:
-      - selector: "#comment-receipt-page div.mb-4 h4"
+      - selector: "#comment-receipt-page h4"
   success:	
-    headers:	
-      status: 200	
-    body:	
+    body:
+      excludes: "Processing, please wait"
```

Comment:
This may end up causing false negatives. If you leave this in you may want to add a hard wait at the end

Reply:
Good call. I decided to just take it out, thought it was cleaner that way